fastlane_bot/helpers/txhelpers.py

```diff
@@ -11,29 +11,22 @@
 import nest_asyncio
 from _decimal import Decimal
 
-# import itertools
-# import random
-# import time
+from json import loads
 from dataclasses import dataclass
 from typing import List, Any, Dict, Optional
 
-# import eth_abi
-# import math
-# import pandas as pd
 import requests
 from alchemy import Network, Alchemy
 from web3.exceptions import TimeExhausted
 
-# from fastlane_bot.config import *  # TODO: PRECISE THE IMPORTS or from .. import config
-# from fastlane_bot.db.models import Token, Pool
-# import fastlane_bot.config as c
-# from fastlane_bot.tools.cpc import ConstantProductCurve
 from fastlane_bot.config import Config
-from fastlane_bot.data.abi import *  # TODO: PRECISE THE IMPORTS or from .. import abi
+from fastlane_bot.data.abi import ERC20_ABI
```

Comment:
may I suggest `from fastlane_bot.data import abi`
and then use it as `abi.ERC20_ABI`

this avoids having to import the ABIs one by one, pollutes the namespace less, and is easier to understand without IDE

Reply:
The comment which was removed (in red) requests the exact opposite.
There's no added value in using `abi.ERC20_ABI` rather than using `ERC20_ABI` across the code.
In fact, there's a little added value in the opposite, as it saves a bit of test.
Anyways, this is minor, so resolving.